UKHomeOffice/nmsw-ui
# NMSW-1082 More information link for UPT in voyages page

# Ticket

NMSW-1082

----

## AC

Given I am on the voyages page
Then there is a link to the ETA help page 

----

## To test

- run app
- sign in
- > you should see a more information section with an ETA link
- click the link
- > It should take you to a gov page with more ETA info

----

## Notes
